ts-tests/tests/bridge.test.ts

```diff
@@ -82,16 +82,24 @@ describeCrossChainTransfer('Test Cross-chain Transfer', ``, (context) => {
         const provider = context.ethConfig.wallets.alice.provider;
         const currentBlock = await provider.getBlockNumber();
         await sleep(15);
+
+        let voteTransactionCount = 0;
```

Comment:
What's this counter for? The loop stops after the first `voteTransaction` anyway, so at the `if (voteTransactionCount === 1) {` part `voteTransactionCount` will _always_ equal 1 :thinking: